frontend/packages/dev-console/integration-tests/features/pipelines/event_tab_pipeline_run_page_ODC_4314

```diff
@@ -0,0 +1,29 @@
+Feature: Event tab in Pipeline run and Task run pages
+	As a user, I want see event tab in Pipeline run and Task run pages
+
+Background:
+    Given Openshift Pipeline operator is installed
+
+@regression
+Scenario: Event tab in pipeline run page
+   Given user is in admin perspective
```

Comment:
I am not sure why it is in admin perspective. It Should be written in the context of Developer Perspective.

Reply:
Admin perspective has a clear division of pages so that's why I wrote it in respect to developer perspective

Reply:
Do you want me to change it or add a new one with respect to developer perspective

Reply:
Change the context to developer perspective as we are writing tests for ODC.